bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -297,26 +328,41 @@ public CloudBigtableScanConfiguration build() {
     private final ValueProvider<String> tableId;
     private final ValueProvider<ReadRowsRequest> request;
     private ReadRowsRequest cachedRequest;
+    private Map<String, ValueProvider<String>> additionalConfiguration;
 
     RequestWithTableNameValueProvider(
         ValueProvider<String> projectId,
         ValueProvider<String> instanceId,
         ValueProvider<String> tableId,
-        ValueProvider<ReadRowsRequest> request) {
+        ValueProvider<ReadRowsRequest> request,
+        Map<String, ValueProvider<String>> additionalConfiguration) {
```

Comment:
Please add `ValueProvider<String> appProfileId` above request, and remove the `additionalConfiguration`